### ISSUE TITLE
[Analyzer Suggestion] Keywords Casing

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#46707]
**Brief description:**

Visual Basic isn't a case-sensitive language, which allows to use the keywords in any casing. While this is perfectly valid, it's much more common to use PascalCasing for keywords.

**Languages applicable:**

Visual Basic

**Code example that the analyzer should report:**

```vb
public class Student
end class
```

should be fixed to:

```vb
Public Class Student
End Class
```

**Comment by sharwell** [comment]
This is a gap between formatting analyzer and the Format Document command for Visual Basic. We should update the formatting analyzer to include this casing check.

**Comment by CyrusNajmabadi** [comment]
Closing out as we are not investing in these areas.
